Test/CoreSDK.Test/Shared/TelemetryClientTest.cs

```diff
@@ -841,7 +841,7 @@ public void SendEventToValidateEndpoint()
 
             HttpClient client = new HttpClient();
             var result = client.PostAsync(
-                "https://dc.services.visualstudio.com/v2/validate",
+                "https://eus-breeze-in.cloudapp.net/v2/validate",
```

Comment:
Oh, forgot to mention. The updated schema is not deployed yet. Thus PR is for separate branch, not develop. I'd like to accumulate all the changes and then push together in develop